declarations/pixiv.json

```
@@ -4,7 +4,10 @@
     "Privacy Policy": {
       "fetch": "https://policies.pixiv.net/en.html#privacy",
```

This now redirects, let's store the redirect target 🙂 
```suggestion
      "fetch": "https://policies.pixiv.net/en/privacy_policy.html",
```